examples/twittermap/web/public/javascripts/map/pointmap.js

```diff
@@ -158,66 +158,69 @@ angular.module('cloudberry.map')
         $scope.scale_x = 0;
         $scope.scale_y = 0;
 
-        //To generate Tweet Popup content from Twitter API (oembed.json?) response JSON
-        function translateOembedTweet(tweetJSON) {
-          var userName = "";
-          try {
-            userName = tweetJSON.author_name;
-          }
-          catch (e){
-            console.log("author_name missing in this Tweet.:" + e.message);
-          }
+        //shows each point's info in Front-end.
+        function translateTweetdatatoShow(tweetJSON) {
+            var userName = "";
+            try {
+                userName = tweetJSON['user.name'];
+            }
+            catch (e){
+                console.log("author_name missing in this Tweet. :" + e.message);
+            }
 
-          var userLink = "";
-          try {
-            userLink = tweetJSON.author_url;
-          }
-          catch (e) {
-            console.log("author_url missing in this Tweet.:" + e.message);
-          }
+            var userPhotoUrl = "";
+            try {
+                userPhotoUrl= tweetJSON['user.profile_image_url'];
+                console.log("userPhotoUrl = " + userPhotoUrl);
+            }
+            catch (e){
+                console.log("user.profile_image_url missing in this Tweet.:");
+            }
 
-          var tweetLink = "";
```

Comment:
We can keep the tweetLink by construct the link using this pattern : https://twitter.com/{username}/status/{tweetid}